specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/disk.json

```diff
@@ -1166,38 +1166,31 @@
             "values": [
               {
                 "value": "Empty",
-                "name": "Empty",
-                "Description": "Create a new disk with a given size."
+                "description": "Create a new disk with a given size."
               },
               {
                 "value": "Attach",
-                "name": "Attach",
-                "Description": "Disk will be attached to a VM."
+                "description": "Disk will be attached to a VM."
               },
               {
                 "value": "FromImage",
-                "name": "From Image",
-                "Description": "Create a new disk by importing from a PIR image or a customer's user image."
+                "description": "Create a new disk by importing from a PIR image or a customer's user image."
               },
               {
                 "value": "Import",
-                "name": "Import",
-                "Description": "Create a new disk by importing from a customer's blob. The blob must be formatted as a VHD."
+                "description": "Create a new disk by importing from a customer's blob. The blob must be formatted as a VHD."
               },
               {
                 "value": "Copy",
-                "name": "Copy",
-                "Description": "Create a new disk by copying from a customer's managed disk or snapshot."
+                "description": "Create a new disk by copying from a customer's managed disk or snapshot."
               },
               {
                 "value": "Restore",
-                "name": "Restore",
-                "Description": "Create a new disk by restoring from a restore point."
+                "description": "Create a new disk by restoring from a restore point."
               },
               {
                 "value": "Upload",
-                "name": "Upload",
-                "Description": "Create a new disk by customer upload using a write SAS to drectly write the contents of their disk to the blob. The contents must be formatted as a VHD."
+                "description": "Create a new disk by customer upload using a write SAS to drectly write the contents of their disk to the blob. The contents must be formatted as a VHD."
```

Comment:
Create a new disk by obtaining a write token and using it to directly upload the contents of the disk